src/components/Popover/Options.js

```diff
@@ -53,11 +53,11 @@ export const Options = ({
 }
 Options.displayName = 'Options'
 Options.propTypes = {
-  activeOption: PropTypes.number,
-  dataKey: PropTypes.string,
-  entities: PropTypes.array,
-  onChange: PropTypes.func,
-  optionsRefs: PropTypes.array,
-  selectedOption: PropTypes.number,
-  setSelectedAndActiveOptions: PropTypes.func,
+  activeOption: PropTypes.number.isRequired,
+  dataKey: PropTypes.string.isRequired,
+  entities: PropTypes.array.isRequired,
+  onChange: PropTypes.func.isRequired,
+  optionsRefs: PropTypes.array.isRequired,
+  selectedOption: PropTypes.number.isRequired,
+  setSelectedAndActiveOptions: PropTypes.func.isRequired,
```

Comment:
My usage(s) use all these props and I don't see it really being functional without them, so I've err'd on side of "is required". I suppose we can back that off in the future if we find use cases like we did for `labelCopy` in other components.